cmake/Dependencies.cmake

```diff
@@ -1377,6 +1377,12 @@ if(USE_DISTRIBUTED AND USE_TENSORPIPE)
     add_subdirectory(${PROJECT_SOURCE_DIR}/third_party/tensorpipe)
 
     list(APPEND Caffe2_DEPENDENCY_LIBS tensorpipe)
+    if(USE_CUDA)
+      list(APPEND Caffe2_CUDA_DEPENDENCY_LIBS tensorpipe_cuda)
+    elseif(USE_ROCM)
+      # Not yet...
```

Comment:
do we need a `message(WARNING, "")` here?

Reply:
Good point I'll add it.